compiler/src/test/scala/org/scalajs/core/compiler/test/JSInteropTest.scala

```diff
@@ -1823,9 +1823,9 @@ class JSInteropTest extends DirectTest with TestHelpers {
         @JSName("foo")
         def a: Int = js.native
       }
-      """ hasWarns
+      """ hasErrors
       """
-        |newSource1.scala:13: warning: A duplicate @JSName annotation is ignored. This will become an error in 1.0.0.
+        |newSource1.scala:13: error: Only a single @JSName annotation per member is allowed.
```

Comment:
"a single [...] per member" is a weird formulation. It's just a feeling, mind you, but I would suggest a formulation similar to that for native load specs, e.g.,

    A member can only have one @JSName annotation.